module5/lessons/Week1/TalkingWhileCoding.md

```diff
@@ -0,0 +1,150 @@
+
+### Tip 3. Think About Your Tone
+
+Try to maintain an enthusiastic approach, even when feeling nervous. You can to to come across as competent and humble, so try to find the middle ground between overconfidence and excessive uncertainty.
+
+### Tip 4. Take a Quiet Pause
+
+Yes, sharing your thought process is important, but if taking a minute to think quietly would help you gather your thoughts that's great too. Just make sure to communicate that with your interviewer instead of just going silent.
+
+
+### Let's Watch an Excellent Example
+
+Individually, make you way to [this](https://www.youtube.com/watch?v=1qw5ITr3k9E) mock technical interview. Watch from 3:30 through 18:08 when she finishes the `Book` class.
+
+While watching, take notes of specific things she says while coding that you think are excellent. When we come back together I'll call on folks to share.
+
+### Your Turn!
+
+You and your partner will each get a chance to be the interviewer and interviewee. 
+
+Whoever is being interviewed will start by opening the appropriate dropdown below, forking the repl, and then sharing their screen so that their interviewer can see their work.
+
+**Schedule:**
+* 15 minutes, Partner 1 answers their question
+* 5 minutes, Partner 2 gives feedback to Partner 1
+* 15 minutes, Partner 2 answers their question
+* 5 minutes, Partner 1 gives feedback to Partner 2
+
+While the interviewer, here are three questions to guide your feedback
+* Did the interviewee explain at least one tradeoff they were making or another option they considered?
+* Did the interviewer share the purpose behind their code?
+* Were the interviewer's explanations easy to follow?
+
+<section class="answer" markdown="1">
+### Partner 1's Question
+
+**Part 1: Is already in [this](https://replit.com/@launch-team/Talking-While-Coding-1) Repl.**
+
+**Part 2: If Partner 1 finishes Part 1 quickly, they can paste the following into their repl for Part 2.**
+
+DESCRIPTION:
+You'll be given a list of two strings, and each will contain exactly one colon (":") in the middle (but not at beginning or end). The length of the strings, before and after the colon, are random.
+
+Your job is to return a list of two strings (in the same order as the original list), but with the characters after each colon swapped.
+
+Examples
+["abc:123", "cde:456"]  -->  ["abc:456", "cde:123"]
+["a:12345", "777:xyz"]  -->  ["a:xyz", "777:12345"]
+
+**They can also paste in the following test cases:**
+
+```c#
+List<string> inputList = new List<string> { "abc:123", "cde:456" };
+List<string> swappedList = SwapStrings(inputList);
+
+//Expected:
+//abc:456
+//cde:123
+foreach (string str in swappedList)
+{
+    Console.WriteLine(str);
+}
+```
+</section>
+
+<section class="answer" markdown="1">
+### Partner 2's Question

```

Comment:
This doesn't appear as a `<h3>` in the render.

Reply:
It doesn't in markdown preview, but when I preview the website it looks good. I think it's because we have markdown="1" as an attribute for the dropdown.